src/bundle_list.c

```diff
@@ -249,15 +250,21 @@ static enum swupd_code show_included_bundles(char *bundle_name, int version)
 	const int MINIMAL_SUBSCRIPTIONS = 2;
 	if (strcmp(bundle_name, "os-core") == 0 || list_len(subs) <= MINIMAL_SUBSCRIPTIONS) {
 		info("\nNo included bundles\n");
-		ret = SWUPD_OK;
+		ret_code = SWUPD_OK;
```

Comment:
ret_code is already SWUPD_OK here.

Reply:
same for other lines setting it to SWUPD_OK after this line

Reply:
no, at this point the ret_code will be `add_sub_NEW` 🤔

Reply:
what line sets ret_code to add_sub_NEW?

Reply:
line 231: `ret = add_subscriptions(bundles, &subs, mom, true, 0);`

Reply:
that's ret and not ret_code

Reply:
🤣 that's true. Man, I've got to pay more attention. I'll make the change.